diff -up fwupd-1.4.5/plugins/uefi/fu-plugin-uefi.c.omv~ fwupd-1.4.5/plugins/uefi/fu-plugin-uefi.c
--- fwupd-1.4.5/plugins/uefi/fu-plugin-uefi.c.omv~	2020-08-15 22:27:08.509640906 +0000
+++ fwupd-1.4.5/plugins/uefi/fu-plugin-uefi.c	2020-08-15 22:27:16.473893270 +0000
@@ -403,7 +403,6 @@ fu_plugin_uefi_load_config (FuPlugin *pl
 	gboolean disable_shim;
 	guint64 sz_reqd = FU_UEFI_COMMON_REQUIRED_ESP_FREE_SPACE;
 	g_autofree gchar *require_esp_free_space = NULL;
-	g_autofree gchar *require_shim_for_sb = NULL;
 	g_autofree gchar *esp_path = NULL;
 
 	/* parse free space needed for ESP */
